src/public/map.js

```diff
@@ -266,6 +290,30 @@ function displayCompanies(companies) {
   }
 }
 
+function displayRegions() {
```

Comment:
The region drop down menu starts off unselected; not drawing the regions on load.